systori/apps/project/api.py

```diff
@@ -105,6 +143,31 @@ def search(self, request):
 
         return JsonResponse({"projects": projects})
 
+    @swagger_auto_schema(method="GET", responses={200: NoteSerializer(many=True)})
+    @swagger_auto_schema(method="POST", request_body=NoteSerializer, responses={201: NoteSerializer()})
+    @action(methods=["GET", "POST"], detail=True)
+    def notes(self, request, pk=None):
+        project = self.get_object()
+        if request.method.lower() == "get":
+            notes = Note.objects.filter(project=project)
+            return Response(data=NoteSerializer(notes, many=True).data)
+
+        if request.method.lower() == "post" and pk:
+            serializer = NoteSerializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+            # Get logged in worker
+            worker = Worker.objects.filter(user=request.user)[0]
```

Comment:
`worker = Worker.objects.get(user=request.user)` should be fine

Reply:
I didn't know about the `get` method. Will update this.